policy.md

```diff
@@ -1062,8 +1063,8 @@ Resources and
 Tools](appendix-resources-tools.md#appendix-resources-and-tools) for
 some sample security questionnaires.
 
-**KEY RECOMMENDATION**: Protecting privacy requires continuous
-attention, and it really starts from project inception.  Invest
+**KEY RECOMMENDATION**: Recognize that protecting privacy and security requires
```

Comment:
Or: "Pay continuous attention to privacy and security from the moment of project inception. ..."